test/test-util.js

```diff
@@ -95,11 +97,12 @@ function testStateTransitions(ns, initial, newState, finalState, incNoDelta, sta
             expectedMembers[ix] = {status: finalState};
 
             // alive is a delta
-            statusCounts.alive = (statusCounts.alive || 0) + n
+            counts = _.extend({}, statusCounts)
```

Comment:
Why is `counts` global?

Honestly, I don't understand this change. What broke and why?